### ISSUE TITLE
Update Pack

### ISSUE THREAD

**Issue by Dijkstra1-zz** [amcoder/curse_client#8]
On the windows curse client packs can be upgraded to latest version leaving changes that the user has mad e.g. configs, added mods. Am completely overlooking in your version if not it would be a nice feature to add.

**Comment by amcoder** [comment]
You're right that the upgrade feature is missing. I have been installing a new instance and copying my world and any changes over manually, but this is a good suggestion to add to the app.

Thanks!
